client/src/screens/BallotScreen.tsx

```diff
@@ -1,77 +1,147 @@
-import React, { useContext } from 'react'
-import { useParams } from 'react-router-dom'
+import React, { useContext, useState } from 'react'
+import { useParams, useHistory } from 'react-router-dom'
 import {
   getBallotStyle,
   getContests,
   getPrecinctById,
+  getElectionLocales,
 } from '@votingworks/ballot-encoder'
 import pluralize from 'pluralize'
 
 import { BallotScreenProps } from '../config/types'
 import AppContext from '../contexts/AppContext'
 
-import Button from '../components/Button'
+import Button, { SegmentedButton } from '../components/Button'
 import HandMarkedPaperBallot from '../components/HandMarkedPaperBallot'
 import { Monospace } from '../components/Text'
-import { getBallotFileName } from '../utils/election'
+import {
+  getBallotFileName,
+  getHumanBallotLanguageFormat,
+} from '../utils/election'
 import NavigationScreen from '../components/NavigationScreen'
 import HorizontalRule from '../components/HorizontalRule'
+import { DEFAULT_LOCALE } from '../config/globals'
```

Comment:
Yeah, I can see how this will make sense for further flexibility for future translation. To keep it simple for now though, I'm gonna just hard-code this value into `AppContext`.

Reply:
On further thought, let's not do this now. Let's do this when we're ready to introduce a mechanism to change the primaryLocaleCode.